dotcom-rendering/src/web/server/showMoreCardsToHtml.tsx

```diff
@@ -0,0 +1,27 @@
+import { renderToString } from 'react-dom/server';
+import { ExtraCardsContainer } from '../components/ExtraCardsContainer';
+
+interface DCRShowMoreContainer {
+	cards: DCRFrontCard[];
+	containerPalette?: DCRContainerPalette;
+}
+/**
+ * cardsToHtml is used by the /Cards endpoint to render extra fronts cards
+ *
+ * @returns string (the html)
+ */
+
+export const cardsToHtml = ({
+	cards,
+	containerPalette,
+}: DCRShowMoreContainer): string => {
+	const html = renderToString(
+		<ExtraCardsContainer
```

Comment:
If I call `/Cards` it seems like the html I get in response is exactly the  `ExtraCardsContainer`?

It can be tempting to write code that is adaptable and open to future use cases but I think in this case it might be clearer if we're more explicit here.

Rather than

Frontend calls `/Cards`
Receives `ExtraCardsContainer`
Inserts it in `ShowMore.importable`

We could have

Frontend calls `/ShowMore`
Receives `ShowMoreContainer`
Inserts it in `ShowMore.importable`

or maybe

Frontend calls `/ShowMore`
Receives `ShowMore`
Inserts it in `EnhanceShowMore.importable` (this follows existing patterns, like `EnhancePinnedPost`)

I think this would really help future developer understand dependencies and what code is related to what.

One use case that was discussed that we thought might have used /Cards was onwards. But - as so often happens - this had it's own special requirements and now we have `/Onwards`.

Reply:
this is a fair point. I think you're right that the original motivation for keeping it more general was the connection with `Onwards`.

My only concern is that `ShowMoreContainer` never contains `ShowMore.importable`, if that makes sense? Would `MoreCardsContainer` work, do you think? Or does that have the same problem as `ExtraCardsContainer`?